src/App.tsx

```diff
@@ -1,223 +1,95 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext, useEffect, useMemo, useRef, useState,
+} from 'react';
+import { getTodos } from './api/todos';
 import { AuthContext } from './components/Auth/AuthContext';
+import { ErrorNotification } from './components/ErrorNotification';
+import { Footer } from './components/Footer';
+import { NewTodo } from './components/NewTodo';
+import { TodoList } from './components/TodoList';
+import { FilterStatus } from './enums/Status';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const user = useContext(AuthContext);
   const newTodoField = useRef<HTMLInputElement>(null);
+  const [userTodos, setUserTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState('');
+  const [filterStatus, setFilterStatus] = (
+    useState<FilterStatus>(FilterStatus.ALL)
```

Comment:
if you set some value of FilterStatus as default then FilterStatus will be default type of state already
```suggestion
    useState(FilterStatus.ALL)
```